README.md

```diff
@@ -1,27 +1,61 @@
 # docker-otel-lgtm
 
-This is work in progress, don't use it.
+[grafana/otel-lgtm](https://hub.docker.com/grafana/otel-lgtm) bundles Grafana's open source stack for OpenTelemetry monitoring in a single Docker image.
 
-## Build and run the Docker image
+![alt](img/overview.png)
 
+## Intended Usage
+
+The intended usage is:
+
+* Quick way to try OSS Grafana with OpenTelemetry.
+* Demos, presentations.
+* Integration tests for OpenTelemetry instrumentation: Run automated queries against the databases to check if expected metrics/traces/logs are present.
+
+The Docker image is not intended for production monitoring.
+
+## Get the Docker image
+
+The Docker image is available on Docker hub: https://hub.docker.com/r/grafana/otel-lgtm
+
+## Run the Docker image
+
+```sh
+docker run -p 3000:3000 -p 4317:4317 --rm -ti grafana/otel-lgtm
+```
+
+## Send OpenTelemetry Data
+
+There's no need to configure anything: The Docker image works with OpenTelemetry's defaults.
+
+```sh
+# Not needed as these are the defaults in OpenTelemetry:
+export OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+export OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
```

Comment:
`service.name` and `service.instance.id` - or is this already too much for this repo?

Reply:
makes sense